src/timer.rs

```diff
@@ -57,6 +57,23 @@ use void::Void;
 /// #     fn wait(&mut self) -> ::nb::Result<(), Void> { Ok(()) }
 /// # }
 /// ```
+///
+/// If you want to use a CountDown timer as an opaque type in a HAL-based driver, you have to add
+/// some type bounds to make sure arguments passed to `start` can be used by the underlying
+/// (concrete) `CountDown::Time` type:
+/// ```rust
+/// extern crate embedded_hal as hal;
+/// #[macro_use(block)]
```

Comment:
```suggestion
```

Reply:
I don't see the block being used anywhere...

Reply:
Heh kind of embarrassing, sorry. This was quick-and-dirty copied out of real code.